src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -97,18 +90,25 @@ const AnonymousInputMessage = ({onSendButtonClicked}: AnonymousInputMessageProps
     return text?.length === 0;
   };
 
+  const sendButtonStyle = React.useCallback(() => {
+    const isDisabled = isDisableButton();
+    if (isDisabled) return colors.gray1;
+    if (type === 'SIGNED') return colors.darkBlue;
+    return colors.bondi_blue;
+  }, [isDisableButton()]);
+
   return (
     <>
       <View style={styles.container}>
         <View style={styles.containerInput}>
           <TextInput multiline style={styles.input} onChangeText={onChangeInput} value={text} />
           <TouchableOpacity
-            style={[styles.btn, isDisableButton() ? styles.disableButton : styles.enableButton]}
+            style={styles.btn}
             disabled={isDisableButton()}
             onPress={handleSendMessage}>
             <IconSend
               style={styles.icSendButton}
-              fillBackground={isDisableButton() ? colors.gray1 : colors.bondi_blue}
+              fillBackground={sendButtonStyle()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of the `sendButtonStyle` function improves the readability of the code by abstracting the logic for determining the button's style. However, it seems like you're using `useCallback` without any dependencies which is unnecessary. You could just use a regular function instead.
```diff
- const sendButtonStyle = React.useCallback(() => {
+ const sendButtonStyle = () => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->